### ISSUE TITLE
Boilerplate

### ISSUE THREAD

**Issue by audiBookning** [benawad/codeponder#5]
I have just seen your latest videos on youtube. They are extremely interesting.
From what i understand, you are making a development environment for creating the codeponder project.

It seems to me that it will have much more value than just for this one project.

When the 'boilerplate' for the development environment is 'finished',  will you consider putting it in a separate repo/package, thus having a viable package to create any other project with it?

**Comment by benawad** [comment]
Yeah I'm going to put it in a separate repo so anyone can reuse it for different projects. 

I was planning on making a cli, but it may be overkill

**Comment by richardtks** [comment]
@benawad, Hi, I thought you have made a simple cli for it already in your day 1 video, but I couldn't find it in your repo though :cry:

**Comment by benawad** [comment]
I haven't combined the cli and boilerplate yet. Not sure when I'm going to get to that, maybe after code ponder.
